File: codeforces/9.helpful_maths/helpful_maths.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  string s;
  cin >> s;
  unsigned short int num_elements = s.size();
  char temp_num;
  for (unsigned short int position = 1; position < ((num_elements + 1) / 2); position++) {
    for (unsigned short int index = 0; index < (num_elements - position); index += 2) {
      if (s[index] > s[index + 2]) {
        temp_num = s[index];
        s[index] = s[index + 2];
        s[index + 2] = temp_num;
      }
    }
  }
  cout << s << endl;
  return 0;
}

File: codeforces/low_difficulty/800_points/9.helpful_maths_[339A]/helpful_maths.py
# Author: José Rodolfo (jric2002)
s = list(input())
num_elements = len(s)
for i in range(1, int((num_elements + 1) / 2)):
  for j in range(0, num_elements - i, 2):
    if (s[j] > s[j + 2]):
      temp_num = s[j]
      s[j] = s[j + 2]
      s[j + 2] = temp_num
print("".join(s))